base/applications/cmdutils/replace/lang/zh-CN.rc

```diff
@@ -0,0 +1,50 @@
+/*
+ * PROJECT:     ReactOS Command shell
```

Comment:
For these translations where "ROS Command Shell" has been copied, please change these to
```suggestion
 * PROJECT:     ReactOS Replace Command
```
From what I see, this should affect the following translations:
pt-BR, ro-RO, ru-RU, zh-CN, zh-TW
and, to a lesser extent, uk-UA

Reply:
Fixed.